src/backend/glutin/mod.rs

```diff
@@ -127,21 +130,24 @@ impl Display {
     pub fn rebuild(
         &self,
         wb: glutin::WindowBuilder,
-        cb: glutin::ContextBuilder,
+        cb: glutin::ContextBuilder<NotCurrent>,
```

Comment:
Should be T, not `NotCurrent`. What if they want to pass a cb which shares with a current context?